### DISCUSSION TITLE
sanitize_html value render form in twig SF 6.4

### DISCUSSION THREAD

**Discussion by ghost** [symfony/symfony#52550] [Q&A]
Hello everyone,

I have a little problem, when I return the content of an entered form, the HTML rendering is impeccable. On the other hand, when I put it in the rendering of a form, I have the html entity which remains. I can put whatever I want. raw, escape, etc... nothing changes.
I removed purify to only work with sanitize. But I'm stuck there :/
Do you have a solution?

![Capture d’écran du 2023-11-11 09-08-19](https://github.com/symfony/symfony/assets/127077166/5214a338-a7eb-412b-97b5-01c2e811fdf7)
![Capture d’écran du 2023-11-11 09-08-55](https://github.com/symfony/symfony/assets/127077166/8aa0b51b-353c-4cef-9430-eaf867a8120a)

Thanks in advance

**Comment by stof** [comment]
This is because you are not applying the filter on the *rendering* of the variable, and so your filter has no effect on the way auto-escaping gets applied (because auto-escaping is applied when rendering the value)

**Comment by ghost** [comment]
Thank you for your response, so I haven't found a solution other than creating a filter without knowing if it will work (a little lack of time). I handed over HtmlPurifierBundle which does the interpretation.
